### DISCUSSION TITLE
Retweet

### DISCUSSION THREAD

**Discussion by Gzcsrkn** [IBAX-io/go-ibax#1704] [Soru Cevap - Türkçe]
Retweet etkinliği sürekli daha sonra tekrarla diyor

**Comment by Bulent3407** [comment]
Twitterdan açıklama yaptılar beklemede kalın diye yapanların puanları silinebilir

**Comment by Maydin38** [comment]
sürekli daha sonra deneyin uyarısı veriyor evet

**Comment by Yusufknk** [comment]
Retweet görevleri için belirli sayıda API kontenjanı var bu kontenjan son retweet görevlerinde az sayıda kaldığı için ilk yapanlar doldurdu. Alınan hataların ve çoğu kişinin yapamamasinin nedeni bu
